src/app/backend/handler/localehandler.go

```diff
@@ -29,12 +30,14 @@ import (
 
 // TODO(floreks): Remove that once new locale codes are supported by the browsers.
 // For backward compatibility only.
-var localeMap = map[string]string{
-	"zh-cn": "zh-Hans",
-	"zh-sg": "zh-Hans-SG",
-	"zh-tw": "zh-Hant",
-	"zh-hk": "zh-Hant-HK",
-}
+// zh-SG, zh-Hans-SG, zh-HK and zh-Hant-HK are not supported by Angular 9 for now.
```

Comment:
Leave the todo about it being a temporary solution and that it should be removed once browsers adopt new language codes.